src/swagger-specs/query-service.yaml

```diff
@@ -1278,38 +1593,72 @@ definitions:
           method:
             type: string
             description: the HTTP method to use
+  
+  PagingProperties:
+    type: object
+    description: Contains properties to describe the order, size, total number of pages, and current page.
+    properties:
+      orderby:
+        type: string
+        description: The property by which this page is ordered.
+        example: '-created'
+      page:
+        type: integer
+        description: The page number of the result.
+        example: 1
+      count:
+        type: integer
+        description: The number of items on the page.
+        example: 26
+      pageSize:
+        type: integer
+        description: The number of items on the page.
+        example: 26
+  
+  version:
+    type: integer
+    description: REST API version of this resource
```

Comment:
What do you mean by the "REST API version" of a resource, as opposed to just a "version"?